.env.example

```diff
@@ -0,0 +1,65 @@
+# Connection URIs
+# ===============
+# These environment variables are used to authenticate
+# API requests. They adopt the connection URI approach
+# from Airflow, which combines various fields (such as
+# access token and API base endpoint) into one string.
+#
+# Legend:
+# - The sections between angle brackets (<foo-bar>) are
+#   placeholders that should be replaced with values.
+#   When replacing values, omit the angle brackets.
+# - The section between square brackets ([/?foo=<bar>])
+#   are optional sections. If they are included, the
+#   placeholders still need to be replaced with values.
+# - Values should be percent/URL-encoded prior to being
+#   included in the connection URI. Do not URL-encode
+#   the entire URI because some special characters must
+#   remain untouched for parsing (such as '@' and ':').
+
+
+# Synapse credentials
+# -------------------
+# <access-token>: Replace with a Synapse personal access
+#   token (PAT), which can be generated here:
+#   https://www.synapse.org/#!PersonalAccessTokens
+#
+# Examples:
+#   SYNAPSE_CONNECTION_URI=syn://:eyJ0[...]QP7g@
+SYNAPSE_CONNECTION_URI=syn://:<access-token>@
```

Comment:
That idea did come up in conversation with @BWMac on Monday. This way, we wouldn't be artificially incrementing Synapse IDs (assuming separate databases), and the files would get deleted automatically (I think). This wouldn't work though for tests that use Synapse files as input unless we create them as part of the setup for the Synapse integration tests. 

To address the fact that Synapse actually has 4 different API endpoints, we can use `connection.host = "staging.synapse.org"` as a shorthand to run [this method](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/c0acef913f51ec99c57031077a9273cc76cbb820/synapseclient/client.py#L302) on the Synapse client.

Reply:
Good idea - I dont think it's urgent to add it but it might be nice for consistency.

The other issue is that the staging endpoint actually isn't always up and running.